src/WebJobs.Extensions/Extensions/Files/Config/FilesJobHostConfigurationExtensions.cs

```diff
@@ -57,16 +59,45 @@ public FilesExtensionConfig(FilesConfiguration filesConfig)
                 _filesConfig = filesConfig;
             }
 
+            private FileInfo GetFileInfo(FileAttribute attribute)
+            {
+                string boundFileName = attribute.Path;
+                string filePath = Path.Combine(_filesConfig.RootPath, boundFileName);
```

Comment:
Should we be checking for null values on the attribute.Path value or the RootPath of the fileconfig?